### ISSUE TITLE
[PITR][Improvement] Good to have keyspace/database name in output JSON of list_snapshot_schedule command.

### ISSUE THREAD

**Issue by Arjun-yb** [yugabyte/yugabyte-db#9615]
Currently **list_snapshot_schedule** command returns one or more schedule lists in JSON format.
Schedule list entries contain:

* schedule ID
* schedule options (interval and retention time)
* a list of snapshots that the system has automatically taken

With these entries it is good to have keyspace/database name, which will be helpful to the users to map schedule ID with keyspace/database easily.

**Comment by tedyu** [comment]
Here is sample output where snapshot JSON shows namespace and table names:
```
                {
                    "namespace": "k",
                    "table": "variant_product_301",
                    "table": "internal_group_id_idx_301",
                    "id": "6112194a-bf62-4c47-ba7b-8fb8cb8747ef",
                    "snapshot_time": "2021-08-06 16:25:48.186966",
                    "previous_snapshot_time": "2021-08-06 16:22:23.321262"
                }
```

**Comment by bmatican** [comment]
Fixed by this backport 9671a34f3047f9abc2148c6b9cb4264cde40330d
